content/courses/tokens-and-nfts/nfts-with-metaplex-core.md

```diff
@@ -0,0 +1,834 @@
+const asset = generateSigner(umi)
+const collection = await fetchCollection(umi, publicKey("CORE_COLLECTION_ADDRESS"));
+
+const { signature, result } = await create(umi, {
+  asset,
+  collection,
+  name: 'My Asset',
+  uri: assetUri,
+}).sendAndConfirm(umi, { send: { commitment: "finalized" } });
+```
+
+**Note**: once you add an Asset to a Collection you'll need to pass in a `CollectionV1` object in 
+the collection field (can be obtained by using the `fetchCollection()` function) of subsequent instructions. 
+Leaving the collection field empty will cause invalidations during transactions causing them to fail.
+
+#### Updating the Asset & Collection
+
+The following steps works for both Assets and Collection, the only difference will be 
+employing the `update` method for Assets and the `updateCollection` method for Collections
+
+The `update` method allows you to update all the fields present in the account like `collection`, 
+`name`, `updateAuthority` and the `uri`
+
+```typescript
+const asset = await fetchAsset(umi, publickey("CORE_ASSET_ADDRESS"));
+const collection = await fetchCollection(umi, publicKey("CORE_ASSET_ADDRESS"));
+
+const { signature, result } = await update(umi, {
+  asset,
+  collection,
+  name: 'My new NFT'
+  uri: newAssetUri
+  newCollection: publicKey("NEW_CORE_COLLECTION_ADDRESS")
+}).sendAndConfirm(umi, { send: { commitment: "finalized" } });
+```
+
+When we add an Asset to a Collection, to not waste any additional bytes, the collection gets stored 
+in the `updateAuthority` field of the asset. So if you want to change the collection address for the 
+asset, you can update the `updateAuthority` field of the `update` function, instead of using the 
+`newCollection` field, like this:
+
+```typescript
+const asset = await fetchAsset(umi, publickey("CORE_ASSET_ADDRESS"));
+const collection = await fetchCollection(umi, publicKey("CORE_ASSET_ADDRESS"));
+
+const { signature, result } = await update(umi, {
+  asset,
+  collection,
+  newUpdateAuthority: updateAuthority("Collection", [publicKey("NEW_CORE_COLLECTION_ADDRESS")])
+}).sendAndConfirm(umi, { send: { commitment: "finalized" } });
+```
+
+**Note**: We can use the same method to add the Asset to a Collection if it isn't part of a 
+collection currently.
+
+A noted difference from the `token-metadata` program is that it not impossible to create an immutable 
+`Asset` using the `create` method. The only way to make an asset immutable is to change the `updateAuthority` 
+to `None` in a futher update.
+
+```typescript
+const asset = await fetchAsset(umi, publickey("CORE_ASSET_ADDRESS"));
+const collection = await fetchCollection(umi, publicKey("CORE_ASSET_ADDRESS"));
+
+const { signature, result } = await update(umi, {
+  asset,
+  collection,
+  newUpdateAuthority: updateAuthority("None")
+}).sendAndConfirm(umi, { send: { commitment: "finalized" } });
+```
+
+**Note**: any fields you don't include in the call to `updateV1` will stay the same, by design.
+
+### Lab
```

Comment:
Didn't you meantion we were scrapping the lab section?

Going to hold off reviewing this section till I know.

Reply:
I think we should leave it here since it's done and letting them ask us to delete if they don't want it in the PR

Reply:
I thought you said thought you said already that they asked us to not add it?